Data/test.lua

```diff
@@ -0,0 +1,9646 @@
+SetSource({
+  SourceLong="Core Rulebook",
+  SourceShort="CR",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy88yj",
+  SourceDate="2009-08",
+})
+DefineClass({
+  HitDie=12,
+  MaxLevel=20,
+  SkillPointsPerLevel=4,
+  ExClass="Ex-Barbarian",
+  Roles={
+    "Combat",
+    "Skill",
+  },
+  Types={
+    "Base",
+    "PC",
+  },
+  Facts={
+    ClassType="PC",
+    Abb="Brb",
+  },
+  Levels={
+    {
+      Level="1",
+      Abilities={
+        {
+          Category="Class",
+          Nature="AUTOMATIC",
+          Name="Barbarian",
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+  },
+  Name="Barbarian",
+  Definitions={
+    {
+      Name="BarbarianLVL",
+      InitialValue=0,
+    },
+  },
+  Bonuses={
+    {
+      Category="COMBAT",
+      Variables={
+        "BASEAB",
+      },
+      Formula="classlevel(\"APPLIEDAS=NONEPIC\")",
```

Comment:
Presumably we're going to want these "formulas" to not be raw strings. But rather actual LUA code... It appears to be another language that would be ideal to translate here.  (Though it's fine if that's a "part 2" feature). :-)

Reply:
I wish they shared more... Like it was "lowSave" or "highSave", and not have to repeat those rules in every single class... but I guess that's fine?

Reply:
Yea, formulas will be done in another PR, they are super strange though.